BuyPolish/Pola/UI/ProductSearch/ReportProblem/ReportProblemViewController.swift

```diff
@@ -69,7 +75,11 @@ final class ReportProblemViewController: UIViewController {
     }
 
     private func updateReportButtonState() {
-        castedView.sendButtom.isEnabled = imageCount > 0
+        if sendImageEnabled {
+            castedView.sendButtom.isEnabled = imageCount > 0
+        } else {
+            castedView.sendButtom.isEnabled = !castedView.descriptionTextView.text.isEmpty
```

Comment:
Zróbmy inaczej. Niech warunkiem będzie `imageCount > 0 || !castedView.descriptionTextView.text.isEmpty` niezależnie od stanu `sendImageEnabled`. Jeśli włączymy możliwość wysyłania zdjęć to nie będą one warunkiem koniecznym do wysłania. Mateusz stwierdził, że nie zawsze się przydają i sam tekst też jest ok.